scripts/diag/rpc-volume-report.py

```
@@ -0,0 +1,181 @@
+            return f"{n:,.1f} {unit}" if unit != "B" else f"{n:,.0f} B"
+        n /= 1024.0
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--last", default="30m")
+    ap.add_argument("--top", type=int, default=12,
+                    help="how many message types to list in each ranking")
+    args = ap.parse_args()
+
+    out = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)